### PR TITLE
Missing site description in messages

### PR DESCRIPTION
**Pull request by shahardo**
#685

### CODE REVIEW

File: negawatt/modules/custom/negawatt_normalizer/negawatt_normalizer.drush.inc [hunk 1]
```diff
@@ -66,6 +70,22 @@ function drush_negawatt_normalizer_reset_last_processed() {
 }
 
 /**
+ * Delete all messages issued.
+ */
+function drush_negawatt_normalizer_delete_messages() {
+  // Fetch all message ids.
+  $query = new EntityFieldQuery();
```
Comment (amitaibu) [inline negawatt/modules/custom/negawatt_normalizer/negawatt_normalizer.drush.inc]: make scalable.

File: negawatt/modules/custom/negawatt_restful/plugins/restful/message/NegawattMessageNotificationBase.php [hunk 2]
```diff
@@ -59,10 +59,18 @@ public function publicFieldsInfo() {
...
 
+    $public_fields['meter_title'] = array(
+      'property' => 'field_message_meter_title',
+    );
+
     $public_fields['place_description'] = array(
       'property' => 'field_message_place_description',
     );
 
+    $public_fields['place_address'] = array(
```
Comment (amitaibu) [inline negawatt/modules/custom/negawatt_restful/plugins/restful/message/NegawattMessageNotificationBase.php]: `address`

File: negawatt/modules/custom/negawatt_restful/plugins/restful/message/NegawattMessageNotificationBase.php [hunk 3]
```diff
@@ -59,10 +59,18 @@ public function publicFieldsInfo() {
       ),
     );
 
+    $public_fields['meter_title'] = array(
+      'property' => 'field_message_meter_title',
+    );
+
```
Comment (amitaibu) [inline negawatt/modules/custom/negawatt_restful/plugins/restful/message/NegawattMessageNotificationBase.php]: `place_description` => `description`

**Comment by shahardo** [comment]
Merged

**Pull request closed without merging**